### DISCUSSION TITLE
The updates come MARCH-ing in 🥁 [Issues Mini Check-in]

### DISCUSSION THREAD

**Discussion by queenofcorgis** [community/community#49640] [Projects and Issues]
Our terrific team has been working hard iterating and iterating on Issues and Projects. The only folks working harder? Our community members as they give feedback, self-solve, and more! Good news - there are plenty of new features that we want to hear your thoughts on below. 

🦸‍♂️ **Incredible in Issues**: A round of applause for these community citizens 👏
* @rasulkireev: Reported a [bug](https://github.com/orgs/community/discussions/46792) 🐞 in Projects complete with screenshots and a recording. Appreciate you flagging this - it is now fixed 🔧. 
* @spenserblack: Came to assist with [Issue Templates](https://github.com/orgs/community/discussions/48397). I personally know how tricky creating those can be so linking the documentation and giving multiple alternatives was very welcome! 

🐙I**ssues Info**: All updates, info, releases, etc that are Issues and Projects-related!
* **Latest Issues & Projects Update**:  We released roadmap markers 🗺️ and command line support for Projects [Learn more](https://github.blog/changelog/2023-03-02-github-issues-projects-march-2nd-update/).  
* **ICYMI**: Back in February, we shipped updates to workflows, roadmaps, our API, and improved cross-organization projects - [check it out](https://github.blog/changelog/2023-02-23-github-issues-projects-february-23rd-update/) .

🧹**Housekeeping**: Community on Discussions updates! 
* **Temporary Interaction Limits on New Accounts**: In an effort to reduce spammy behavior, we are instituting temporary interaction limits to bar accounts less than 24hrs-old from participating in the github.com/community Discussions space. [Read the notice](https://github.com/orgs/community/discussions/48334). 

See you next month! What update are you most excited about?

**Comment by ghost** [comment]
It sounds like there have been some great updates and improvements to Issues and Projects on Github, and the community has been actively involved in providing feedback and reporting bugs. It's great to see that Github is responsive to this feedback and is continuously iterating to improve their platform. Some of the updates mentioned include roadmap markers and command line support for Projects, as well as updates to workflows, roadmaps, and the API. Additionally, there have been some housekeeping updates to the community on Discussions, such as temporary interaction limits for new accounts to reduce spam. It's exciting to see what updates and improvements Github will bring next!

**Comment by Poopooracoocoo** [comment]
Not excited about any of this. Please let repo maintainers limit who can create new issues, who can create discussions, and who can create PRs. 
The temporary functionality isn't good enough. So many open source projects have the issue of flooded issues and it makes development difficult as there isn't a dedicated space for confirmed and properly written bug reports or feature requests.
